assets/fonts/fonts.js

```diff
@@ -0,0 +1,25 @@
+require('./GHGuardianHeadline-Bold.woff');
```

Comment:
does this mean everyone wil have to download all fonts?

Reply:
actaully - what does this do? i'm not sure?

Reply:
In order to tell webpack to include the fonts in the `compile-assets`, we provide this file as an entry point.

Reply:
We do that in [this line](https://github.com/guardian/support-frontend/pull/634/files#diff-5c572bb8f24a00a84aa28f22d3a39ba5R24).